### ISSUE TITLE
Move demo app to sample data repos

### ISSUE THREAD

**Issue by vpicavet** [IGNF/itowns-legacy#11]
It is currently broken.

The idea would be that all demos use by default a prebuilt version of iTowns, available at itowns.github.io/itowns, which is the gh_pages branch of itowns.

As for now the prebuilt lib should be manually pushed to the repo.
Afterwards we can setup auto-build & push on new release tag.

To solve the current situation, it seems like we need : 
- a new itowns build in the gh_pages branch with latest addition from mbredif ( #10 )
- update index.html for both demos in the sample data repos

@tbroyer sounds right ? Could you push the new build ?

**Comment by tbroyer** [comment]
1. pushed `dist/itowns.js` to `gh-pages` (removed everything else)
2. moved `index.html` to itowns-sample-data repository with inlined `DemoUI.js` and linking to "local data" (note: I had to slightly change/fix the build to avoid bundling jQuery, but that means you now need to load jQuery before you load iTowns, as with many other libraries that depend on jQuery; maybe we should do the same with Three.js BTW, but that's another story)
3. copied the `index.html` into itowns.github.io as `v1demo.html`, using data from itowns-sample-data and images from itowns-sample-data-small, and updated the site to load that file instead of the previous demo from this repo's `gh-pages`.

I also updated the `index.html` in itowns-sample-data-small even though it's currently broken (it was already broken anyway)

Next step is to update the README and the website.
